functions/default.pp

```diff
@@ -8,8 +8,7 @@ function postgresql::default(
 ) {
   include postgresql::params
 
-  #search for the variable name in params first
-  #then fall back to globals if not found
-  pick( getvar("postgresql::params::${parameter_name}"),
-  "postgresql::globals::${parameter_name}")
+  # Search for the variable name in params.
+  # params inherits from globals, so it will also catch these variables.
+  pick(getvar("postgresql::params::${parameter_name}"))
```

Comment:
Late to the party, but what's the point of using `pick()` with a single parameter? Isn't just `getvar()` the same?

Reply:
`getvar('nothing_here')` returns `undef`.  Since pick was already there, I assumed we wanted to have a strong failure with an exception in that case.

Reply:
I suspected that, but good to hear that explicitly.